packages/dx-chart-core/src/utils/event-tracker.ts

```diff
@@ -39,7 +40,7 @@ const buildEventHandler = (seriesList: SeriesList, handlers: HandlerFnList): Eve
     hitTesters = hitTesters || createHitTesters();
     const targets: TrackerTarget[] = [];
     seriesList.forEach(({ name: series, index: order, symbolName }) => {
-      const status = hitTesters![symbolName](location);
+      const status = hitTesters![symbolName as unknown as string](location);
```

Comment:
Looks like `string` should be replaced with `symbol` in `HitTesters` declartion.

Reply:
There is ts error: Type 'symbol' cannot be used as an index type.